cmd/metrictank/metrictank.go

```diff
@@ -59,6 +60,7 @@ var (
 
 	// Data:
 	dropFirstChunk    = flag.Bool("drop-first-chunk", false, "forego persisting of first received (and typically incomplete) chunk")
+	ingestAfterStr    = flag.String("ingest-after", "", "discards data from an org id received before the next chunk boundary after the specified unix timestamp (in seconds since 1 Jan 1970); syntax: org-id:timestamp")
```

Comment:
1) this is not true in case the specified timestamp is a chunk boundary (than it is not the next chunk)
2) the description of what a unix timestamp is incomplete (UTC timezone), but also this is not the place to explain a unix timestamp
3) would it make sense to have multiple rules for multiple orgs? I thought it did
4) the message is clunky. can we simplify it?

Reply:
>     1. this is not true in case the specified timestamp is a chunk boundary (than it is not the next chunk)

I thought it was the case given how `ingestAfterNextChunkT0` is computed in `aggmetric.go`:

`ingestAfterNextChunkT0 = uint32(ingestAfter-(ingestAfter%int64(ret.ChunkSpan))) + ret.ChunkSpan`

> 
>     2. the description of what a unix timestamp is incomplete (UTC timezone), but also this is not the place to explain a unix timestamp

Removed.

> 
>     3. would it make sense to have multiple rules for multiple orgs (eg as csv) ? I thought it did

Implemented (546e9aa).

> 
>     4. the message is clunky. can we simplify it?

I took a stab at simplifying it, let me know what you think (689b147).

Reply:
> I thought it was the case given how ingestAfterNextChunkT0 is computed in aggmetric.go:

you are correct!

However, I imagine the typical use for a flag like this will be to pass a chunk boundary timestamp, so something like:
```
~ ❯❯❯ TZ=UTC date -d '06:00'
Wed 07 Aug 2019 06:00:00 AM UTC
~ ❯❯❯ TZ=UTC date -d '06:00' +%s
1565157600
~ ❯❯❯ TZ=UTC date -d 'tomorrow 00:00'
Thu 08 Aug 2019 12:00:00 AM UTC
~ ❯❯❯ TZ=UTC date -d 'tomorrow 00:00' +%s
1565222400
```

I don't imagine people would pass a value such as 
```
~ ❯❯❯ TZ=UTC date -d '23:59:59' +%s
1565222399
```

Thus, assuming this hunch is correct and people would pass a clean chunk boundary, i would argue it's confusing if ingestion is only for data that is 1 chunk "after" the timestamp i specified.
Thus perhaps instead of the formula you use, we should use `mdata.AggBoundary()` and change the name "ingest-after" to "ingest-from", describing that is is to ingest data "from this timestamp and onwards" rathar than strictly "after".

WDYT?

Reply:
> I took a stab at simplifying it, let me know what you think (689b147).

 :+1:

Reply:
> > I thought it was the case given how ingestAfterNextChunkT0 is computed in aggmetric.go:
> 
> you are correct!
> 
> However, I imagine the typical use for a flag like this will be to pass a chunk boundary timestamp (or, a common enough use), so something like:
> 
> ```
> ~ ❯❯❯ TZ=UTC date -d '06:00'
> Wed 07 Aug 2019 06:00:00 AM UTC
> ~ ❯❯❯ TZ=UTC date -d '06:00' +%s
> 1565157600
> ~ ❯❯❯ TZ=UTC date -d 'tomorrow 00:00'
> Thu 08 Aug 2019 12:00:00 AM UTC
> ~ ❯❯❯ TZ=UTC date -d 'tomorrow 00:00' +%s
> 1565222400
> ```
> 
> I don't imagine people would often pass a value such as
> 
> ```
> ~ ❯❯❯ TZ=UTC date -d '23:59:59' +%s
> 1565222399
> ```
> 
> Thus, assuming this hunch is correct and people would commonly pass a clean chunk boundary, i would argue it's confusing if ingestion is only for data that is 1 chunk "after" the timestamp i specified.
> Thus perhaps instead of the formula you use, we should use [`mdata.AggBoundary()`](https://godoc.org/github.com/grafana/metrictank/mdata#AggBoundary) and change the name "ingest-after" to "ingest-from", describing that is is to ingest data "from this timestamp and onwards" rathar than strictly "after".
> 
> WDYT?


Implemented with commits 67e3d6a and 3a498f4.